src/test/java/rest/salesforce/org/steps/ApiSteps.java

```diff
@@ -0,0 +1,233 @@
+        apiResponseObject = apiResponse.getBody(ApiResponseObject.class);
+    }
+
+    @After("@DeleteAccount")
+    public void deleteAccount() {
+        requestBuilder
+                .clearPathParams()
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", apiResponseObject.getId())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        ApiManager.execute(requestBuilder.build());
+    }
+
+    @After("@DeleteAsset")
+    public void deleteAsset() {
+        requestBuilder
+                .clearPathParams()
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addEndpoint("/Asset/{assetID}")
+                .addPathParams("assetID", apiResponseObject.getId())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        ApiManager.execute(requestBuilder.build());
+    }
+
+    @After("@DeleteContact")
+    public void deleteContact() {
+        requestBuilder
+                .clearPathParams()
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addEndpoint("/Contact/{contactID}")
+                .addPathParams("contactID", apiResponseObject.getId())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        ApiManager.execute(requestBuilder.build());
+    }
+
+    @After("@DeleteContract")
+    public void deleteContract() {
+        requestBuilder
+                .clearPathParams()
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addEndpoint("/Contract/{contractID}")
+                .addPathParams("contractID", apiResponseObject.getId())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        ApiManager.execute(requestBuilder.build());
+    }
+
+    @After("@DeleteOrder")
+    public void deleteOrder() {
+        requestBuilder
+                .clearPathParams()
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addEndpoint("/Order/{orderID}")
+                .addPathParams("orderID", apiResponseObject.getId())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        ApiManager.execute(requestBuilder.build());
+    }
+
+    @Given("I build a {string} request")
+    public void iBuildARequest(final String apiMethod) {
+        requestBuilder.addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addMethod(ApiMethod.valueOf(apiMethod));
+    }
+
+    @When("^I create body with parameters$")
+    public void iCreateBodyWithParameters(final Map<String, String> entry) throws JsonProcessingException, ParseException {
```

Comment:
this method is long maybe you can divide in submethods

Reply:
I will  change the implementation of this step, but due to the lack of time I will make it on my next PR.